eva-lib/src/main/java/uk/ac/ebi/eva/lib/spring/data/entity/StudyBrowser.java

```diff
@@ -0,0 +1,280 @@
+package uk.ac.ebi.eva.lib.spring.data.entity;
+
+
+import uk.ac.ebi.eva.lib.models.VariantStudy;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Created by jorizci on 03/10/16.
+ */
+@Entity
+@Table(name = "study_browser")
+public class StudyBrowser {
+
+    @Id
+    @Column(name = "project_accession")
+    private String projectAccession;
+
+    @Column(name = "studyId")
+    private long studyId;
+
+    @Column(name = "projectTitle")
+    private String projectTitle;
+
+    private String description;
```

Comment:
This comes from a view without any limitation, all strings are defined as TEXT technically I should define here everything as Lob I just refrained as practically none of the elements are really text and if I use lob they will be handled as big text with lazy loading by default.